src/box/errcode.h

```diff
@@ -141,287 +141,313 @@ struct errcode_record {
-	_(ER_TRIGGER_EXISTS, 55,		"Trigger '%s' already exists") \
-	_(ER_USER_MAX, 56,			"A limit on the total number of users has been reached: %u") \
-	_(ER_NO_SUCH_ENGINE, 57,		"Space engine '%s' does not exist") \
-	_(ER_RELOAD_CFG, 58,			"Can't set option '%s' dynamically") \
-	_(ER_CFG, 59,				"Incorrect value for option '%s': %s") \
+	_(ER_MULTISTATEMENT_TRANSACTION, 54,	"Can not perform %s in a multi-statement transaction", "operation", STRING) \
+	_(ER_TRIGGER_EXISTS, 55,		"Trigger '%s' already exists", "trigger", STRING) \
+	_(ER_USER_MAX, 56,			"A limit on the total number of users has been reached: %u", "max", UINT) \
+	_(ER_NO_SUCH_ENGINE, 57,		"Space engine '%s' does not exist", "engine", STRING) \
+	_(ER_RELOAD_CFG, 58,			"Can't set option '%s' dynamically", "option", STRING) \
+	_(ER_CFG, 59,				"Incorrect value for option '%s': %s", "option", STRING, "details", STRING) \
 	_(ER_SAVEPOINT_EMPTY_TX, 60,		"Can not set a savepoint in an empty transaction") \
 	_(ER_NO_SUCH_SAVEPOINT, 61,		"Can not rollback to savepoint: the savepoint does not exist") \
-	_(ER_UNKNOWN_REPLICA, 62,		"Replica %s is not registered with replica set %s") \
-	_(ER_REPLICASET_UUID_MISMATCH, 63,	"Replica set UUID mismatch: expected %s, got %s") \
-	_(ER_INVALID_UUID, 64,			"Invalid UUID: %s") \
+	_(ER_UNKNOWN_REPLICA, 62,		"Replica %s is not registered with replica set %s", "replica", STRING, "replicaset", STRING) \
+	_(ER_REPLICASET_UUID_MISMATCH, 63,	"Replica set UUID mismatch: expected %s, got %s", "expected", STRING, "actual", STRING) \
+	_(ER_INVALID_UUID, 64,			"Invalid UUID: %s", "uuid", STRING) \
 	_(ER_REPLICASET_UUID_IS_RO, 65,		"Can't reset replica set UUID: it is already assigned") \
-	_(ER_INSTANCE_UUID_MISMATCH, 66,	"Instance UUID mismatch: expected %s, got %s") \
-	_(ER_REPLICA_ID_IS_RESERVED, 67,	"Can't initialize replica id with a reserved value %u") \
-	_(ER_INVALID_ORDER, 68,			"Invalid LSN order for instance %u: previous LSN = %llu, new lsn = %llu") \
-	_(ER_MISSING_REQUEST_FIELD, 69,		"Missing mandatory field '%s' in request") \
-	_(ER_IDENTIFIER, 70,			"Invalid identifier '%s' (expected printable symbols only or it is too long)") \
-	_(ER_DROP_FUNCTION, 71,			"Can't drop function %u: %s") \
-	_(ER_ITERATOR_TYPE, 72,			"Unknown iterator type '%s'") \
-	_(ER_REPLICA_MAX, 73,			"Replica count limit reached: %u") \
-	_(ER_INVALID_XLOG, 74,			"Failed to read xlog: %lld") \
-	_(ER_INVALID_XLOG_NAME, 75,		"Invalid xlog name: expected %lld got %lld") \
-	_(ER_INVALID_XLOG_ORDER, 76,		"Invalid xlog order: %lld and %lld") \
+	_(ER_INSTANCE_UUID_MISMATCH, 66,	"Instance UUID mismatch: expected %s, got %s", "expected", STRING, "actual", STRING) \
+	_(ER_REPLICA_ID_IS_RESERVED, 67,	"Can't initialize replica id with a reserved value %u", "value", UINT) \
+	/* ER_INVALID_ORDER, 68, Unused */ \
+	_(ER_MISSING_REQUEST_FIELD, 69,		"Missing mandatory field '%s' in request", "field", STRING) \
```

Comment:
"value" would probably bebetter - we use "field" to denote tuple fields.

Reply:
Fixed.

```diff
+       _(ER_MISSING_REQUEST_FIELD, 69,         "Missing mandatory field '%s' in request", "value", STRING) \
```